src/tools/sdk/index.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Dart SDK
+description: Dart libraries and command-line tools.
+permalink: /tools/sdk
+js:
+- url: /tools/sdk/archive/assets/install.js
+  defer: true
+---
+
+The Dart SDK has the libraries and command-line tools that you need to develop
+Dart apps. To learn about other tools you can use for Dart development, see
+[Dart Tools](/tools).
+
+The [DartPad][]{: target="_blank"} is a great way to learn how to write a simple
+app in the Dart language, once you are ready to develop real apps, you need the
+Dart SDK.
+
+This site's documentation and examples use the
+[{{site.data.pkg-vers.SDK.vers}}][site SDK version]{:.no-automatic-external}
+release of the **Dart SDK**.
+
+## Install the SDK
+
+A package manager can help you easily install and update the Dart SDK.
+
```

Comment:
Let's put the archive link here:

Alternatively, you can download zip files from the [Index of Downloads].

Reply:
I've added both the archive and "build from source" links here.